iotedgedev/solution.py

```diff
@@ -26,3 +30,14 @@ def create(self, name):
         self.output.footer("Azure IoT Edge Solution Created")
         if name != "":
             self.output.info("Execute 'cd {0}' to navigate to your new solution.".format(name))
+
+    def is_dir_empty(name):
```

Comment:
Please put this in Utility.py as we may need it in other places later.

Can you also create tests for these use cases?